src/compiler/proposals-and-stabilization.md

```diff
@@ -263,11 +263,20 @@ circumstance.
     in the description
   - New targets should be assigned to the compiler team co-leads to ensure that any demands on
     the project infrastructure are considered and checked with relevant teams
+- Renaming a target or making a breaking change to a target
```

Comment:
Remark: note that a precedent for renaming a *Tier 2* target is https://github.com/rust-lang/compiler-team/issues/607, where it's proposed using an MCP and approved using Second + FCP. In addition, the `wasm-wasi` Tier 2 targets were also renamed with a carefully executed migration plan that helps to mitigate ecosystem breakages by carefully announcing intention in advance, spending a release cycle to warn, and *then* drop the original target tuple.

Breaking changes to targets might depend on its tier. For instance, I don't think it makes sense to require FCP for tier 3 targets.

Reply:
This is useful context, updated to reflect precedent.